src/ducks/apps/index.js

```diff
@@ -1,18 +0,0 @@
-import * as fromCozyClient from 'redux-cozy-client'
```

Comment:
Let's keep in mind this commit, this part could still be useful to get Drive URL, unless we also use interapp redirection for it :)